lib/svg4everybody.js

```diff
@@ -0,0 +1,114 @@
+/*! svg4everybody v1.0.0 | github.com/jonathantneal/svg4everybody */
+
+function embed(svg, g) {
+	if (g) {
+		var viewBox = !svg.getAttribute('viewBox') && g.getAttribute('viewBox');
+		var fragment = document.createDocumentFragment();
+		var clone = g.cloneNode(true);
+
+		if (viewBox) {
+			svg.setAttribute('viewBox', viewBox);
+		}
+
+		while (clone.childNodes.length) {
+			fragment.appendChild(clone.childNodes[0]);
+		}
+
+		svg.appendChild(fragment);
+	}
+}
+
+function loadreadystatechange(xhr) {
+	xhr.onreadystatechange = function () {
+		if (xhr.readyState === 4) {
+			var x = document.createElement('x');
+
+			x.innerHTML = xhr.responseText;
+
+			xhr.s.splice(0).map(function (array) {
+				embed(array[0], x.querySelector('#' + array[1].replace(/(\W)/g, '\\$1')));
+			});
+		}
+	};
+
+	xhr.onreadystatechange();
+}
+
+function svg4everybody(opts) {
+	opts = typeof opts === 'object' ? opts : {};
+
+	var uses = document.getElementsByTagName('use');
+	var script = document.createElement('script');
+	var fallbackSrc = 'fallbackSrc' in opts ? opts.fallbackSrc : function (src) {
+		return src.replace(/\?[^#]+/, '').replace('#', '.').replace(/^\./, '') + '.png' + (/\?[^#]+/.exec(src) || [''])[0];
+	};
+	var shim = false;
+	var shimExternalContent = 'shimExternalContent' in opts ? opts.shimExternalContent : /\bEdge\/12\b|\bTrident\/[567]\b|\bVersion\/7.0\s+Safari\b/.test(navigator.userAgent) || (navigator.userAgent.match(/AppleWebKit\/(\d+)/) || [])[1] < 537;
+	var requestAnimationFrame = window.requestAnimationFrame || setTimeout;
+	var svgCache = {};
+
+	if (LEGACY_SUPPORT && 'shim' in opts) {
```

Comment:
I tried both because I was hoping it would solve an issue that turned out to be something else. If lib one is not supposed to be used directly without LEGACY_SUPPORT anyway, you can ignore my note.